src/agent/transform-headers-agent.ts

```diff
@@ -124,11 +136,11 @@ class TransformHeadersAgent extends WrappedAgent {
      * @param {string[]} sortedHeaders - array of headers in order
      * @returns {Function} - sort function
      */
-    createSort(sortedHeaders) {
-        const sortWithSortedHeaders = (a, b) => this.sort(a, b, sortedHeaders);
+    createSort(sortedHeaders: string[]) {
+        const sortWithSortedHeaders = (a: string, b: string) => this.sort(a, b, sortedHeaders);
 
         return sortWithSortedHeaders;
     }
 }
 
-module.exports = TransformHeadersAgent;
+export default TransformHeadersAgent;
```

Comment:
I think we're not using default exports in the packages migrated to TypeScript. Named export would be better.